hathor/p2p/sync_v2/agent.py

```diff
@@ -151,8 +165,8 @@ def get_status(self) -> dict[str, Any]:
         """
         res = {
             'is_enabled': self.is_sync_enabled(),
-            'peer_height': self.peer_height,
-            'synced_height': self.synced_height,
+            'peer_best_block': self.peer_best_block.to_json() if self.peer_best_block else None,
+            'synced_block': self.synced_block.to_json() if self.synced_block else None,
```

Comment:
Although the change is good, we should probably create an issue to update the monitoring tools, which I believe already implemented support for the current API. Maybe even keeping the current fields alongside the new ones before completely removing them.

Reply:
We can easily keep the old format. I think this format is weird because the height doesn't tell the whole story. I mean, the fact that two peers have the height does not mean they are synced.

Reply:
Yes, but it gives a good idea on the sync progress, from an external point of view, if this API only returns the block's hash, and we don't have that block, we'd need to query the node again on another API to learn that block's height.

Reply:
The current change returns both the block id and the height.

Reply:
Should I add the hashes on separate fields? That's another way to go here and keep compatibility.

Reply:
Oh, right it's a block-info, not a hash. I think it's good as is, the monitoring should be able to catch up to this change.